### PR TITLE
Scrape - birds

### PR DESCRIPTION
**Pull request by cassiemak**

### CODE REVIEW

File: lib/tasks/get_birds.rake [hunk 1]
```diff
@@ -1,18 +1,67 @@
...
+    # puts birds 
+
+    endOfList = false
+    begOfList = false
+    # scrape the li an loop thru it with .each
+    birds.each do |bird|
+      # puts the li
+      if endOfList == false
+        bird_common_name = bird.css('a').attr('title').text.squish
+        if bird_common_name == "Pacific loon" then begOfList = true end
```
Comment (harrychenca) [inline lib/tasks/get_birds.rake]: hard to read

File: lib/tasks/get_birds.rake [hunk 2]
```diff
@@ -1,18 +1,67 @@
...
+    birdsFormat = "div > div.mw-content-ltr > ul > li"
+    birds = html_doc.css(birdsFormat)
+    # puts birds 
+
+    endOfList = false
+    begOfList = false
+    # scrape the li an loop thru it with .each
+    birds.each do |bird|
+      # puts the li
+      if endOfList == false
```
Comment (h8rry) [inline lib/tasks/get_birds.rake]:
``` ruby

bird_common_name = bird.css('a').attr('title').text.squish

case bird_common_name
when "Pacific loon"
  begOfList = true
when "List of birds"
  endOfList = true
end

if endOfList == false and begOfList == true
  // scrape
end



```

File: lib/tasks/scrape_gov_trails.rake [hunk 3]
```diff
@@ -1,7 +1,7 @@
 namespace :scrape_gov_trails do 
   desc "Scrape trail's info from http://www.lcsd.gov.hk/en/healthy/hiking/"
 
-  task :get_trails => :trails do
+  task :get_trails => :environment do
```
Comment (h8rry) [inline lib/tasks/scrape_gov_trails.rake:4]: This is not done right? Why are we merging this?

Comment (cassiemak) [inline lib/tasks/scrape_gov_trails.rake:4]: This is not done. that was just a skeleton. but we might not need it at all

**Pull request merged**